Classes/OpenidService.php

```diff
@@ -76,6 +78,9 @@ class OpenidService extends AbstractService
      */
     protected static $openIDLibrariesIncluded = false;
 
+    /** @var \Psr\Log\LoggerInterface */
+    protected $logger;
```

Comment:
I suggest using the LoggerAwareInterface and the according trait instead of defining that manually

Reply:
Done.